lyaemu/flux_power.py

```diff
@@ -113,13 +75,14 @@ def __init__(self, numlos = 32000, max_z= 4.2, max_k = 5.):
         #and hence does not commute with mean flux rescaling.
         #I have checked that for k < 0.1 the corrected version
         #is identical to the unsmoothed version (without mean flux rescaling).
-        self.spec_res = 0.
+        self.spec_res = 127
         #For BOSS the pixel resolution is actually 69 km/s.
         #We use a much smaller pixel resolution so that the window functions
         #are small for mean flux rescaling, and also so that HCDs are confined.
-        self.pix_res = 10.
+        self.pix_res = 123.4
```

Comment:
The pix_res has to be smaller than the spec_res: spec_res is a Gaussian smoothing, pix_res is the size of a pixel. Smoothing over a window size < 1 pixel doesn't make sense.